modules/odata/odata-core/src/main/java/org/eclipse/dirigible/engine/odata2/sql/processor/AbstractSQLProcessor.java

```diff
@@ -58,16 +58,17 @@
 public abstract class AbstractSQLProcessor extends ODataSingleProcessor implements SQLProcessor {
 
     private static final Logger logger = LoggerFactory.getLogger(AbstractSQLProcessor.class);
+    private static final String DATASOURCE_CONTEXT_KEY = "DATASOURCE";
 
     private final OData2EventHandler odata2EventHandler;
     private DataSource dataSource;
     private final ResultSetReader resultSetReader;
 
-    private static final String DUMMY_BUILDER = "dummyBuilder";
-    private static final String SELECT_BUILDER = "selectBuilder";
-    private static final String INSERT_BUILDER = "insertBuilder";
-    private static final String UPDATE_BUILDER = "updateBuilder";
-    private static final String SQL_CONTEXT = "sqlContext";
+    private static final String DUMMY_BUILDER_CONTEXT_KEY = "dummyBuilder";

```

Comment:
can we reverse DUMMY_BUILDER_CONTEXT_KEY -> CONTEXT_KEY_DUMMY_BUILDER to become common prefix, not common suffix

Reply:
we can, but the idea to be a suffix is that you can easily type it and autocomplete it and in the same time you have the information where this constant is used - as a key in the context. But if you prefer prefix, I am also fine with it.